src/plugins/hetero/plugin.cpp

```diff
@@ -162,7 +174,19 @@ QueryNetworkResult Engine::QueryNetwork(const CNNNetwork& network, const Configs
 }
 
 Parameter Engine::GetMetric(const std::string& name, const std::map<std::string, Parameter>& options) const {
-    if (METRIC_KEY(SUPPORTED_METRICS) == name) {
+    if (ov::supported_properties == name) {
+        return decltype(ov::supported_properties)::value_type{
+            ov::PropertyName{ov::supported_properties.name(), ov::PropertyMutability::RO},
+            ov::PropertyName{ov::device::full_name.name(), ov::PropertyMutability::RO},
+            ov::PropertyName{ov::device::architecture.name(), ov::PropertyMutability::RO},
+            ov::PropertyName{ov::device::capabilities.name(), ov::PropertyMutability::RO},
+            ov::PropertyName{ov::device::priorities.name(), ov::PropertyMutability::RW},
+            // Properties below are propogated to hw plugin
```

Comment:
do we need these properties?
I supposed HETERO should report only its own properties, while hints etc will be passed via `ov::device::properties(CPU, hint1)`, is not it? Otherwise, we are washing away the boundaries between HW / virtual properties and their responsibilities.

Reply:
> do we need these properties? I supposed HETERO should report only its own properties, while hints etc will be passed via `ov::device::properties(CPU, hint1)`, is not it? Otherwise, we are washing away the boundaries between HW / virtual properties and their responsibilities.

I think these common properties should be supported by all plugins

1. User experience. Instead of specifying ov::device::properties(DEVICE1, hint1),ov::device::properties(DEVICE2, hint1),ov::device::properties(DEVICE3, hint1) we can specify it at once. Also switching between different devices requires minimal changes in application.
2. It is aligned with other virtual devices.
3. We can avoid modification of tools and samples specially for HETERO case.

Reply:
Agreed to fix get_supported_property to provide ov::device::priorities to virtual devices, so they can provide supported properties even for HW plugins.
For this, we also need to adjust virtual devices itself.